test/client/fixtures/sandbox/iframe-test.js

```diff
@@ -294,6 +294,32 @@ test("'body.appendChild' method works incorrectly in the particular case (GH-421
     checkIframes();
 });
 
+test('hammerhead should be initialized after the document.write call in iframe with same-domain src (GH-1325)', function () {
+    return createTestIframe({ src: getSameDomainPageUrl('../../data/iframe/simple-iframe.html') })
+        .then(function (iframe) {
+            var stringifiedNativeWriteFn = nativeMethods.documentWrite.toString();
```

Comment:
Drop this line and appropriate assertions. You already check that iframe is properly initialized using link's src.